pkg/packagekit/kpatch.jsx

```diff
@@ -233,7 +233,7 @@ export class KpatchSettings extends React.Component {
 
         if (this.state.loaded === false || this.state.patchName === null) {
             // Not yet recognized
-            state = <Spinner isSVG size="md" />;
+            state = <Spinner size="md" />;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230425-151143-725ccf60-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)